src/App.js

```diff
@@ -39,8 +39,13 @@ class App extends Component {
     console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
+  onItemClicked = (itemType) => {
+    console.log(itemType)
+    if (itemType === "litter") {
+      this.setState({
+        point: this.state.points += 1
```

Comment:
This should be `points: this.state.points + 1`, plural, not singular, and no assignment.